backend/protos/xyz/block/ftl/v1/language/language.proto

```diff
@@ -34,6 +34,9 @@ message ModuleConfig {
   // LanguageConfig contains any metadata specific to a specific language.
   // These are stored in the ftl.toml file under the same key as the language (eg: "go", "java")
   google.protobuf.Struct language_config = 10;
+
+  // The directory containing the SQL migration files
+  string sql_migration_dir = 11;
```

Comment:
What's the syntax of the directory? `<sql_migration_dir>/<database_name/*.sql` ?

Reply:
No docs yet, but none of the DB stuff has docs yet.